Packs/DemistoLocking/Integrations/DemistoLock/DemistoLock.yml

```diff
@@ -31,7 +31,6 @@ script:
       description: Additional information to provide for the lock instance
     - name: timeout
       description: Timeout (seconds) for wait on lock to be freed
-      defaultValue: "600"
```

Comment:
why did you remove that ?

Reply:
It was overriding the timeout parameter

Reply:
not sure i follow, this is the timeout parameter

Reply:
we have a timeout config parameter set for the instance, and a timeout argument sent for the getlock command. It doesnt really make sense to have a default on the argument, since the default should be taken from the config parameter. Having a default on the argument overrode the config parameter

Reply:
meaning the timeout argument always had a value, so the config param was ignored